src/LightStep/HighLevel/IO.hs

```diff
@@ -28,6 +28,15 @@ import System.Timeout
 globalSharedMutableSpanStacks :: MVar (HM.HashMap ThreadId [Span])
 globalSharedMutableSpanStacks = unsafePerformIO (newMVar mempty)
 
+data LogEntryKey =  ErrorKind | Event | Message | Stack | Custom String
```

Comment:
Why not `Custom T.Text` since it must be converted to Text anyway?

Reply:
Indeed, changed!